3529/2/telegram.rb

```diff
@@ -0,0 +1,28 @@
+require 'telegram/bot'
+require 'yaml'
+require 'httparty'
+require 'open-uri'
+require 'logger'
+
+Dir[File.dirname(__FILE__) + '/Lib/*.rb'].each { |file| require file }
+
+class WebhooksController < Telegram::Bot::UpdatesController
+  include ContextHandler
+  include Telegram::Bot::UpdatesController::MessageContext
+
+  def initialize(_one, _two)
+    super
+    Telegram::Bot::UpdatesController.session_store = :redis_store, { expires_in: 100_000_000 }
+  end
+end
+
+file_token = YAML.safe_load(File.read('Data/token.yaml'))
```

Comment:
can we move all this logic bellow to the class and then call it from 1 file like `MySuppaSweetBot.start/run` ?